### DISCUSSION TITLE
Test discussion

### DISCUSSION THREAD

**Discussion by jonmeow** [carbon-language/carbon-lang#427] [General]
Just a test discussion for people to see and look at the feature, versus Discourse.

**Comment by zygoloid** [comment]
Test reply.

**Reply by jonmeow** [reply]
Reply to reply. :)

**Reply by chandlerc** [reply]
> Test reply.

Further reply, with implicit nesting by quoting, but flattened into a single reply thread.

**Comment by jonmeow** [comment]
Another reply (kind of interested in ordering)

**Comment by jonmeow** [comment]
#444 is working on this...

**Comment by jonmeow** [comment]
Hmmm, #427?
